test_settings.py

```diff
@@ -103,7 +103,7 @@ def root(*args):
         'user': '50/minute',
         'service_user': '60/minute',
     },
-
+    'DATETIME_FORMAT': '%Y-%m-%dT%H:%M:%SZ',
```

Comment:
Our dates are now all tz-aware, UTC dates, so it's ok to explicitly specify the terminating `Z`.  Note that the default format is `iso-8601`, which renders the timezone offset as `+00:00`.  The [terminating `Z` is also accepted under `iso-8601`](https://en.wikipedia.org/wiki/ISO_8601#UTC), but python's `isoformat()` function prefers the explicit numeric tz offset.

This will also guard against the changes in Django REST framework 3.4, which [uses millisecond precision by default](http://www.django-rest-framework.org/topics/3.4-announcement/#microsecond-precision-when-returning-time-or-datetime).